### DISCUSSION TITLE
Suggestion: use the WordPress Bulk Edit feature

### DISCUSSION THREAD

**Discussion by kokogit** [publishpress/publishpress-taxonomies#1519] [Feature Requests]
Greetings from the UK.
My sympathies for what France has suffered over the last days.

I LOVE your Simple Tag plugin.
You just saved me hours of tedious work.

I’ve been editing over 500 custom posts (Film and Game reviews) and I used the Tag field to record my progress in the editing task, so I could list the posts and see at-a-glance which had been edited, which needed to be edited next.

I thought that was a good idea and it worked well until I finished the editing and was ready to make the site public. Before publication I needed to delete all my editing progress tags.

Then I found that you cannot do that with the WP bulk edit or quick edit feature! I had to go to the full edit screen for 500+ posts to delete or change all the tags! Oh NO!!!

Then I found Simple Tags. I was delighted that it works with custom posts and taxonomies. I was able to edit all 500 posts in about 20 minutes.

Thank you so much for that!

You asked for suggestions for improvement. Well the task could have taken me about 5 minutes if you combine the WP bulk edit feature with your system.
I can already search all my posts for, say tag = “edited”. 

![screen shot 2015-11-26 at 11 25 06](https://cloud.githubusercontent.com/assets/2184559/11422202/f468c13e-9431-11e5-92ec-2b3e4ed49277.png)

Next I’d like to Select All (or select by individual tick boxes).
Then select the actions Edit.

![screen shot 2015-11-26 at 11 25 17](https://cloud.githubusercontent.com/assets/2184559/11422198/e867e1da-9431-11e5-942b-cb840f1bccd2.png)

Then delete the existing tag or type my replacement tag ONCE and click UPDATE ALL

I hope you find the suggestion helpful.

Keep up the good work.

Best Wishes

Robert

Robert Hart Fletcher
robertHF@me.com

**Comment by stevejburge** [comment]
Related: https://github.com/TaxoPress/taxopress/issues/43
